.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow should be triggered when a release is done (workflow_call)
+# And also need to be able to run manually (workflow_dispatch). It uses
+# the current version from poetry as a tag for the docker container and 
+# pushes the container to ghcr.
+
+name: "Publish to GHCR"
```

Comment:
is GHCR a known acronym? I know what it means, but shouldn't we write it out for clarity